docker/development/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
+services:
+  dev_dca:
+    build:
+      context: ../../
+      dockerfile: docker/development/Dockerfile
```

Comment:
Is er een speciale reden waarom dit bestand weg verstopt zit in een subdirectory? Zou m.i. prima in project root kunnen.

Reply:
Nee geen specifieke reden hiervoor, ben het zelf gewend om dit altijd zo te doen in projecten waar ik in werk waar we alles in 1 docker directory werken ipv alles in de root te plaatsen. In deze docker dir plaatsen we dan ook aanvullende files zoals bv config bestanden (php.ini of een settings.php in geval van Drupal).
Het zou hier inderdaad in de app root kunnen staan, al ben ik wel geneigd om alles in een directory Docker te plaatsen voor het overzicht.